k8s/upgrade/src/constant.rs

```diff
@@ -70,7 +70,7 @@ pub(crate) const HELM_RELEASE_NAME_LABEL: &str = "openebs.io/release";
 
 pub(crate) const DEFAULT_IMAGE_REGISTRY: &str = "docker.io";
 /// Upgrade containers to develop.
-pub(crate) const UPGRADE_JOB_TO_DEVELOP_TAG: &str = "develop";
+pub(crate) const UPGRADE_JOB_TO_DEVELOP_TAG: &str = "release-2.1";
```

Comment:
Why is the variable named as "to develop"?

Reply:
Used a more generic name for the variable.